src/browserless.ts

```diff
@@ -117,6 +119,14 @@ export class Browserless extends EventEmitter {
       new Router(this.config, this.browserManager, this.limiter, this.Logger);
   }
 
+  private loadPwVersions = async (): Promise<void> => {
```

Comment:
Since we can extend these in downstream SDK Projects, I always tend to go for `protected` members versus `private`